cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -166,11 +166,11 @@ func (w *wizard) doCLIOpt(choice string, options []wizardOpt) {
 	log.Error("That's not something I can do")
 }
 
-var RinkebyNetworkId = "4"
-var DevenvNetworkId = "54321"
+var RinkebychainID = "4"
```

Comment:
Capitalize the "C" in chainID? Think that would be easier to read i.e. `RinkebyChainID` and `DevenvChainID`